README.md

```diff
@@ -6,15 +6,159 @@
 
 This is the unofficial Loopring Python Image Generator and Minter on Layer 2.
 
-⚠️ **NEVER SHARE YOUR PRIVATE KEY OR API KEY WITH ANYONE** ⚠️
-
-## New Wiki
-
-1. Python to generate images and metadata according to the standards for the new Layer 2 marketplace.
-2. NodeJS to pre-calculate CIDs on images and metadata files.
-3. Python to run the batch minting and transfer processes.
-4. PHP to tie everything together.
+⚠️ **NEVER SHARE YOUR PRIVATE KEYS OR PASSPHRASES WITH ANYONE** ⚠️
 
 Checkout the [Wiki pages](https://loopygen.ezl2.app) for up to date info on using the app!
 
 Checkout the [issues section](https://github.com/sk33z3r/LooPyGen/issues) to report any problems, make feature requests, or keep up with our progress!
+

```

Comment:
Add a short _LooPyGen UI_ section that points to the wiki setup page. Also a message similar to "LooPyGen UI is what most creators will want to use, follow this link for the guides"

Reply:
Added and tweaked